web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -161,7 +167,10 @@ internal fun Route.behandlingRoutes(
     }
 
     patch("$behandlingPath/{behandlingId}/iverksett") {
-        // TODO authorize attestant
+        if (!call.erAttestant()) {
```

Comment:
Tenker at denne typen if-sjekk er greit nok inntil videre. Kan hende vi blir nødt til/bør lage "noe mer" etterhvert.

Reply:
Det er vel heller ikke noe i veien for å komme med et litt mer beskrivende feilsvar? Vi kunne kanskje lagret Respons-typen i et companion object (vi har vel en ErrorJson someplace) som vi og kunne brukt.

Reply:
@hestad det var så långt mina norska-skillz kunde gå. 😄